src/components/inputs/rich-text-input/editor.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Button from './button';
+import StyleDropdown from './style-dropdown';
+import MarkButton from './buttons/mark-button';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+import useToggleState from '../../../hooks/use-toggle-state';
+import {
+  BoldIcon,
+  ItalicIcon,
+  UnorderedListIcon,
+  OrderedListIcon,
+  UnderlineIcon,
+  RedoIcon,
+} from './icons';
```

Comment:
Are we going to keep these new icons separate from the existing ones?

Reply:
Yeah, at least for now